airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/write/bulk_loader/BigQueryBulkLoader.kt

```diff
@@ -82,9 +82,11 @@ class BigQueryBulkLoader(
             "Finished loading data into table ${tableId.toPrettyString()}. ${stats.outputRows} rows loaded; ${stats.badRecords} bad records."
         }
         if (stats.badRecords > 0) {
-            logger.warn {
+            // This should be impossible: the load job uses the default setting of maxBadRecords=0,
+            // so the job is supposed to fail if there were any bad records.
+            throw RuntimeException(
```

Comment:
could be convinced to throw ConfigErrorException here. Mostly not sure if this is (a) worth retrying on, and (b) worth paging for

Reply:
I think we should page since we don't expect this to happen.